formatter.go

```diff
@@ -428,11 +449,15 @@ func (f *DefaultFormatter) fillDelta(
 
 func (f *DefaultFormatter) formatValue(value interface{}) string {
 	if flt := extractFloat32(value); flt != nil {
-		return f.formatFloatValue(*flt, 32)
+		return f.addDigitGrouping(f.formatFloatValue(*flt, 32))
 	}
 
 	if flt := extractFloat64(value); flt != nil {
-		return f.formatFloatValue(*flt, 64)
+		return f.addDigitGrouping(f.formatFloatValue(*flt, 64))
+	}
+
+	if intg := extractInt(value); intg != nil {
+		return f.addDigitGrouping(strconv.Itoa(*intg))
 	}
```

Comment:
Here we will catch only int, but we should handle other integers types too: uint, int8, int16, etc.

The easiest way to do it is:

* check if isNumber returns true; it means that it's an integer, because floats are already handled above
* if yes, format it using fmt.Sprintf with %v - this should be equivalent to how we format integers below
* apply digit groupping to that string, and return it